examples/object-type-inclusion/object_type_inclusion.md

```diff
@@ -0,0 +1,7 @@
+## Object type inclusion
+
+Type inclusion enables to create an object type that includes another object type, such that one interface extends another interface and, creates a class that includes a type, like the class implementing the interface. Using the `*T` notation, you can include the object type to another object type or another class.
```

Comment:
> Type inclusion enables to create an object type that includes another object type, such that one interface extends another interface and, creates a class that includes a type, like the class implementing the interface.

This sentence seems a bit too long. Shall we break it into maybe 2 sentences? It'll be easier to understand then.

Reply:
> Using the `*T` notation, you can include the object type to another object type or another class.

Shall we just say 
> You can include object types using the `*T` syntax.

here. What we have now, seems to be repeating the same thing in the previous sentence.

Reply:
Shall we also include

> The implementation of the object type within the class that includes the type is checked at the compile time. This provides interface inheritance. Ballerina does not support implementation inheritance.

Reply:
Updated 9d5526315210351e8e92f1c0d36238e5ae574152